docs/development/dapr-metrics.md

```diff
@@ -83,6 +83,11 @@ Dapr uses prometheus process and go collectors by default.
 * dapr_runtime_actor_deactivated_total: The number of the successful actor deactivation.
 * dapr_runtime_actor_deactivated_failed_total: The number of the failed actor deactivation.
 
+#### Resiliency
+
+* dapr_runtime_resiliency_loaded: The number of resiliency policies loaded.
+* dapr_runtime_resiliency_count: The number of times a resiliency policy has been executed.
```

Comment:
Would prefer 
dapr_runtime_resiliency_exec_count instead of just count

Reply:
We can look into changing this for 1.10